server/prisma/datamodel.graphql

```diff
@@ -127,4 +127,6 @@ type Configuration {
   slackChannelHook: String
 
   tags: Json
+
+  workplaceButton: Boolean! @default(value:false)
```

Comment:
`enableWorkplaceSharing` or `workplaceSharingEnabled` would be clearer I think.

Reply:
Good